package.json

```diff
@@ -193,7 +193,8 @@
     "react-redux": "5.0.6",
     "react-router": "2.8.1",
     "react-router-scroll": "0.4.3",
-    "react-textarea-autosize": "^5.1.0",
+    "react-super-responsive-table": "0.3.0",
+    "react-textarea-autosize": "5.1.0",
```

Comment:
We should probably have a test for this similar to our devDependencies test that *ensures* we use the caret.